### PR TITLE
feat: first user become admin, only if kratos enabled

### PR DESCRIPTION
**Pull request by Effet**
#### What type of this PR

Add one of the following kinds:
/kind feature

#### What this PR does / why we need it:
work with kratos, make first user as admin

#### Which issue(s) this PR fixes:
Fixes #

#### Specified Reviewers:
/assign @sfwn

### CODE REVIEW

File: modules/cmdb/services/member/member.go [hunk 1]
```diff
@@ -728,9 +728,9 @@ func (m *Member) IsAdmin(userID string) bool {
 		}
 	}
 	if member.ID == 0 {
-		logrus.Warnf("CAUTION: user(%s) current no admin, will become soon", userID)
+		logrus.Warnf("CAUTION: user(%s) currently not an admin, will become soon if no one admin exist", userID)
 		// TODO: some risk
-		if m.noOneAdmin() && len(userID) > 11 {
+		if len(userID) > 11 && m.noOneAdminForKratos() { // len > 11 imply that is kratos user
```
Comment (sfwn) [inline modules/cmdb/services/member/member.go:733]: why 11

Comment (Effet) [reply modules/cmdb/services/member/member.go:733]: just a magic value, kratos' user_id is UUID, it is significantly larger than 11

Comment (sfwn) [reply modules/cmdb/services/member/member.go:733]: please comment it clearly for latter coders.

Comment (sfwn) [reply modules/cmdb/services/member/member.go:733]: please comment it clearly for latter coders.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/cmdb/services/member/member.go (hunk 1 above)
Comment (Effet) [reply modules/cmdb/services/member/member.go:734]: you're right, added

**Review by sfwn — APPROVED** [review]

**Pull request merged**